.bumpversion.cfg

```diff
@@ -4,6 +4,4 @@ commit = True
 tag = True
 tag_name = {new_version}
 
-[bumpversion:file:setup.py]
-search = version="{current_version}",
-replace = version="{new_version}",
+[bumpversion:file:scrapy_zyte_api/__version__.py]
```

Comment:
I've checked this locally - after running bump2version - two files were changed with the new version:
__version__.py (version was changed)
.bumpversion.cfg (current_version was changed)
Looks good.
Thanks for the suggestion! @wRAR

Reply:
@kmike @BurnzZ could you take a look at the more or less final version?

Reply:
For the record, the reason the prior config had 2 additional settings was because it involved `setup.py`, where there was a chance other version numbers (those of dependencies) would accidentally get upgraded if they ever matched the package version. That is avoided by forcing the whole `version="…"` pattern to be searched for in `setup.py`, instead of just the version number. That’s not needed for `__version__.py`, though.